### PR TITLE
[8.0][FIX] business_requirement_deliverable_project

### PR DESCRIPTION
**Pull request by martintorresvm**
Fix open issue regards:
https://github.com/OCA/business-requirement/issues/95
On relations One2Many need to be careful using structures
like [6, 0, [ids]]

### CODE REVIEW

**Comment by martintorresvm** [comment]
@elicoidal @seb-elico

File: business_requirement_deliverable_project/wizard/generate_projects.py [hunk 1]
```diff
@@ -166,9 +166,9 @@ def _prepare_project_vals(self, br, parent):
             description = br.description
             privacy_visibility = br.project_id.privacy_visibility \
                 or br.project_id._defaults['privacy_visibility']
-            br_id = br.id
-        elif br._name == 'business.requirement.deliverable':
-            br_id = br.business_requirement_id.id
+        #     br_id = br.id
```
Comment (elicoidal) [inline business_requirement_deliverable_project/wizard/generate_projects.py]: @victormartinelicocorp 
Your PR looks strange: you comment removed lines.
I would say: remove dead code, do not comment.
Something I missed?

Comment (martintorresvm) [reply business_requirement_deliverable_project/wizard/generate_projects.py]: @elicoidal no you are not missing a thing, sorry but this is dead code. Already I commit to fix it

**Review by elicoidal — APPROVED** [review]

**Comment by elicoidal** [comment]
@pedrobaeza @dreispt I think this one is good to go

**Comment by pedrobaeza** [comment]
I don't get the problem that this PR solves.

**Comment by martintorresvm** [comment]
> I don't get the problem that this PR solves.

@pedrobaeza This is a functional problem.

We like to keep the `project_id` as it's created the BR and the previous code overwrite with `linked_project` when generates project on the wizard.

cc @elicoidal

**Comment by pedrobaeza** [comment]
But this one does nothing with linked_project, but with field br_ids...

**Comment by martintorresvm** [comment]
@pedrobaeza It's a relation is one2many on `project.project` so the real `id` is on BR object.

When create a project if I pass the values `br_ids` mean that I will overwrite the BR `project_id` value with the `id` of the project I am creating. 

So indirectly when create the linked project I am replacing the id of project_id on the BR and later I assign to `linked_project` the value which is what is real intend to do.

**Review by pedrobaeza — APPROVED** [review]
OK, thanks for the explanations.

**Pull request merged**